# header: the frame_id inside is for the final target map frame.
#         In current adlink_ddsbot ver. it is unfunctional. 
# robots: the container of robots

std_msgs/Header header
adlink_ddsbot/Robot[] robots

